termine/announce.go

```diff
@@ -36,7 +36,7 @@ var (
 func init() {
 	cmdAnnounce.Run = RunAnnounce
 	cmdAnnounce.Flag.StringVar(&targetmailaddr, "address", "treff@noname-ev.de", "Mailadresse, an die Ankündigungen gehen sollen.")
-	cmdAnnounce.Flag.BoolVar(&confirmAnnounce, "confirm", false, "Frage nach Bestätigung bevor die Mail gesendet wird")
+	cmdAnnounce.Flag.BoolVar(&confirmAnnounce, "confirm", true, "Frage nach Bestätigung bevor die Mail gesendet wird")
```

Comment:
`termine announce` is specifically intended to be run regularly without user intervention by a timer, not just for Stammtisch, but also announcing each c1/4h with speaker and topic each week. The -confirm-flag is there for testing purposes, as is the target-mailaddress, therefore I would leave this as it is and instead concentrate on fixing outstanding issues with this command instead so that we can actually bring it into automated scheduling.